app/controllers/service_areas_controller.rb

```diff
@@ -2,17 +2,21 @@ class ServiceAreasController < ApplicationController
   before_action :set_service_area, only: [:show, :edit, :update, :destroy]
 
   def index
-    @service_areas = ServiceArea.all
+    @service_areas = ServiceArea.order(ServiceArea.arel_table[:name].lower.asc)
```

Comment:
Cool, TIL!

Reply:
it would be cooler if it seemed to be working! lol. it's the "right" code afaict, but doesn't seem to be sorting correctly

Reply:
@exbinary whoa. just figured it out. once we include `Mobility`, it's no longer storing ANY data in the table's field on which we've enabled mobility (so, `pluck(:name)` == `[nil, nil, nil...]`)